server/src/main/java/com/flipkart/varadhi/web/Extensions.java

```diff
@@ -98,5 +102,19 @@ public static <T> T getApiResponse(RoutingContext ctx) {
         public static void todo(RoutingContext context) {
             throw new NotImplementedException("Not Implemented.");
         }
+
+        public static String getResourceIdFromPath(RoutingContext ctx, ResourceType resourceType) {
```

Comment:
as suggested elsewhere, this method can be moved to AuthzHandler only.

Reply:
done